### ISSUE TITLE
Waluigi's Taco Stand no sombrero

### ISSUE THREAD

**Issue by anonymous7002** [gonetz/GLideN64#2231]
In Waluigi's Taco Stand, a SM64 rom hack, Waluigi collects ingredients to tacos with a sombrero in his head.
With GlideN64, Waluigi doesn't have a sombrero, but his normal cap instead.
It's a strange bug.

**Comment by loganmc10** [comment]
Does it work on a real N64 or in angrylion?

**Comment by anonymous7002** [comment]
Yes, in both.
But angrylion is really slow.
That's probably because of my bad integrated Intel GPU.

**Comment by anonymous7002** [comment]
GlideN64
![GLideN64_WALUIGIS_TACO_STAND_000](https://user-images.githubusercontent.com/57062013/80403493-56d72b00-88c8-11ea-8b25-3d9d8a7e33cd.png)
Jabo's Direct3D8
![jabo](https://user-images.githubusercontent.com/57062013/80403428-3c04b680-88c8-11ea-93fe-a55eebc1b966.png)

**Comment by anonymous7002** [comment]
The sombrero is an object and afaik Gliden64 doesn't render some custom objects, while other plugins do.

**Comment by gonetz** [comment]
I need the rom and savestate.

**Comment by anonymous7002** [comment]
I can't give you a rom, but I can give you a patch file which you can apply with Floating IPS.
You need vanilla SM64 US rom in z64 format.
Also, you don't need a savestate, because Waluigi has sombrero on (or doesn't have) almost the whole game.
[Waluigi's Taco Stand 64.zip](https://github.com/gonetz/GLideN64/files/4550983/Waluigi.s.Taco.Stand.64.zip)

**Comment by anonymous7002** [comment]
If you try to play it on Project64 2.x, the emulator crashes once you stomp a goomba.
To fix it, you need to go to Recompiler under the game’s settings and uncheck Advanced block linking.

**Comment by anonymous7002** [comment]
Have you tested it yet?

**Comment by gonetz** [comment]
No. There are plenty of other tasks, where I already can load a rom, a savestate and see the problem right away.

**Comment by anonymous7002** [comment]
[Waluigi's Taco Stand.zip](https://github.com/gonetz/GLideN64/files/5761982/Waluigi.s.Taco.Stand.zip)

**Comment by anonymous7002** [comment]
Testing with mupen64plus revealed this to be an HLE bug.
With paraLLEl-RSP, the sombrero rendered correctly.
